Sources/TokamakCore/Fiber/FiberReconciler+TreeReducer.swift

```diff
@@ -57,9 +57,61 @@ extension FiberReconciler {
       }
     }
 
+    static func reduce<S>(into partialResult: inout Result, nextScene: S) where S: Scene {
+      var result = partialResult
+      Self.reduce(
+        into: &result,
+        nextValue: nextScene,
+        createFiber: { scene, elementParent, _ in
+          Fiber(
+            &scene,
+            parent: partialResult.fiber,
+            element: partialResult.nextExistingAlternate?.element,
+            elementParent: elementParent,
+            environment: nil,
+            reconciler: partialResult.fiber?.reconciler
+          )
+        },
+        update: { fiber, scene, _ in
+          fiber.update(with: &scene)
+        },
+        visitChildren: { $0._visitChildren }
+      )
+      partialResult = result
+    }
+
     static func reduce<V>(into partialResult: inout Result, nextView: V) where V: View {
+      var result = partialResult
+      Self.reduce(
+        into: &result,
```

Comment:
Is there a need to copy to temporary `result` here? Can't `partialResult` be passed directly?

Reply:
`createFiber` accesses it, and mutation requires exclusive access, so I have to make a copy AFAIK. But I could also rework `createFiber` to pass all the parameters in.